.github/workflows/smoke.yml

```diff
@@ -21,12 +19,15 @@ jobs:
           chmod -R 777 ~/.penumbra/testnet_data
           docker-compose build
 
-      - name: Configure local wallet.
-        run: cargo run --bin pcli -- wallet import-from-phrase "$SEED_PHRASE" &> /dev/null
-        env:
-          SEED_PHRASE: ${{ secrets.TEST_SEED_PHRASE }}
-
       - name: Run testnet for smoke test duration.
         run: timeout --preserve-status $TESTNET_RUNTIME docker-compose up --exit-code-from pd-node0
         env:
-          TESTNET_RUNTIME: 4m
+          TESTNET_RUNTIME: 5m
+
+      - name: Now start testnet again in the background so we can run integration tests.
+        run: docker-compose up --detach pd-node0
+
+      - name: Run integration tests against localhost.
+        run: cargo test --package pcli -- --ignored
+        env:
+          PENUMBRA_NODE: 192.167.10.10
```

Comment:
Rather than restarting the smoke-test testnet and running the integration tests against it, could we just add the `cargo test` invocation as a post deployment step on the testnet deployment actions?

That way, we'd ensure that the deployment actions fail if the test fails, and we don't need a separate action run.

Reply:
Oh, NVM, I see that this is changed to run on _every_ push, that seems better, and obsoletes my comment!

 I think it would still be good to add to the deployment actions, though.